api/taxonomies/serializers.py

```diff
@@ -28,16 +31,85 @@ class TaxonomizableSerializerMixin(ser.Serializer):
         'subjects',
     ])
 
-    subjects = ser.SerializerMethodField()
+    def __init__(self, *args, **kwargs):
+        super(TaxonomizableSerializerMixin, self).__init__(*args, **kwargs)
+        request = kwargs['context']['request']
+
+        if self.expect_subjects_as_relationships(request):
+            subject_kwargs = {
+                'related_view': self.subjects_related_view,
+                'related_view_kwargs': self.subjects_view_kwargs,
+                'read_only': False,
+                'many': True,
+                'required': False,
+            }
+
+            if self.subjects_self_view:
+                subject_kwargs['self_view'] = self.subjects_self_view
+                subject_kwargs['self_view_kwargs'] = self.subjects_view_kwargs
+
+            self.fields['subjects'] = RelationshipField(**subject_kwargs)
+        else:
+            self.fields['subjects'] = ser.SerializerMethodField()
+
+    @property
+    def subjects_related_view(self):
+        """
+        For building the subjects RelationshipField on __init__
+        for later API versions only
+        """
+        raise NotImplementedError()
+
+    @property
+    def subjects_view_kwargs(self):
+        """
+        For building the subjects RelationshipField on __init__
+        for later API versions only
+        """
+        raise NotImplementedError
```

Comment:
Add subjects_related_view and subjects_view_kwargs to your view so your subject relationship can be built properly.